src/components/Modal/BaseModal.tsx

```diff
@@ -61,15 +49,15 @@ function BaseModal({
 
     /**
      * Hides modal
-     * @param {Boolean} [callHideCallback=true] Should we call the onModalHide callback
+     * @param callHideCallback - Should we call the onModalHide callback
      */
     const hideModal = useCallback(
         (callHideCallback = true) => {
             if (shouldSetModalVisibility) {
                 Modal.setModalVisibility(false);
             }
             if (callHideCallback) {
-                onModalHide();
+                onModalHide?.();
```

Comment:
You defined default prop - in taht case conditional calling is needed?

Reply:
I don't think so, updated!